test/Microsoft.DocAsCode.Build.UniversalReference.Tests/UniversalReferenceDocumentProcessorTest.cs

```diff
@@ -55,9 +57,52 @@ public void ProcessPythonReferencesShouldSucceed()
             {
                 var outputRawModelPath = GetRawModelFilePath(fileName);
                 Assert.True(File.Exists(outputRawModelPath));
+                var rawModel = File.ReadAllText(outputRawModelPath);
```

Comment:
put in the following UT. should remove this line...

---
In reply to: [114244755](https://github.com/dotnet/docfx/pull/1618#discussion_r114244755) [](ancestors = 114244755)